core/src/main/resources/messages/messages.properties

```diff
@@ -33,6 +33,8 @@ error.user.exists.created.on.buddy.request=User with mobile number ''{0}'' alrea
 error.user.exists=User with mobile number ''{0}'' already exists
 error.user.maximum.number.reached=Reached the maximum number of users that is currently allowed
 
+error.number.not.whitelisted=Mobile number ''{0}'' does not occur in the whitelist of allowed numbers
```

Comment:
Please change it this way: "Mobile number ''{0}'' does not occur on the white list of allowed numbers"